docs/en/ingest-management/processors/processor-add_kubernetes_metadata.asciidoc

```diff
@@ -0,0 +1,226 @@
+[[add_kubernetes_metadata-processor]]
+= Add Kubernetes metadata
+
+++++
+<titleabbrev>add_kubernetes_metadata</titleabbrev>
+++++
+
+// REVIEWERS: Is there really a situation where users need to configure this
+// processor?
+// If so, what are the implications on default indexers and matchers? Do they
+// exist, or does the user always need to define them if they are adding this
+// processor?
+
+// Also I think the field names need to be updated here. What should they be?
```

Comment:
What are the field names mentioned here?

Reply:
After a month on PTO, I honestly can't remember, so I'm marking this as resolved.